lib/utils/isValidIdentifier.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+/**
+ * returns whether a string is a valid CSS identifier
+ * (i.e. only alphanumeric characters, `-`, and `_`;
+ * does not have a leading digit, leading dash followed by digit, or two leading dashes)
+ * furthermore, any escaped or ISO 10646 characters are allowed.
+ * see: https://www.w3.org/TR/CSS2/syndata.html#value-def-identifier
```

Comment:
[trivial] The JSDoc tag has better IDE support.

```suggestion
 * @see https://www.w3.org/TR/CSS2/syndata.html#value-def-identifier
```